composer/loggers/logger_hparams.py

```diff
@@ -215,3 +216,27 @@ class TQDMLoggerBackendHparams(BaseLoggerBackendHparams):
     def initialize_object(self, config: Optional[Dict[str, Any]] = None) -> TQDMLoggerBackend:
         from composer.loggers.tqdm_logger import TQDMLoggerBackend
         return TQDMLoggerBackend(config=config)
+
+
+@dataclass
+class MosaicMLLoggerBackendHparams(BaseLoggerBackendHparams):
+    """:class:`~composer.loggers.mosaicml_logger.MosaicMLLoggerBackend`
+    hyperparameters.
+
+    See :class:`~composer.loggers.mosaicml_logger.MosaicMLLoggerBackend`
+    for documentation.
+    """
+    job_id: int = hp.required("The id of the job to write logs for.")
+    sweep_id: int = hp.required("The id of the sweep to write logs for.")
+    creds_file: Optional[int] = hp.optional(
```

Comment:
YAHP has automatic loading of environment variables. It's totally undocumented, but if you have an env variable with the same name as the cli flag, then YAHP will read it.

Reply:
should this be an int?